docs/guides/installing_concord_termux.md

```diff
@@ -0,0 +1,39 @@
+# Installing Concord on Termux
+
+## Termux? What's that?
+
+Termux is a terminal emulator and a Linux environment app, that emulates the Linux environment giving you some Linux commands and a lot of Linux features.
```

Comment:
Termux is a terminal emulator and a Linux environment app for Android devices. It contains a Linux environment, giving you some Linux commands and a lot of Linux features (such as a C compiler).